### ISSUE TITLE
buildah 1.11.3-2 fails on every command

### ISSUE THREAD

**Issue by maysunfaisal** [containers/buildah#1923]
<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->

**Steps to reproduce the issue:**
1. Download rpm from `https://download-ib01.fedoraproject.org/pub/fedora/linux/updates/30/Everything/x86_64/Packages/b/buildah-1.11.3-2.fc30.x86_64.rpm`
2. Install buildah using yum
```
yum -y --enablerepo=epel install buildah.rpm
Loaded plugins: fastestmirror, ovl
Examining buildah.rpm: buildah-1.11.3-2.fc30.x86_64
Marking buildah.rpm as an update to buildah-1.11.2-2.git0bafbfe.fc30.x86_64
Resolving Dependencies
--> Running transaction check
---> Package buildah.x86_64 0:1.11.2-2.git0bafbfe.fc30 will be updated
---> Package buildah.x86_64 0:1.11.3-2.fc30 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

==============================================================================================================================================================================================================================================
 Package                                                 Arch                                                   Version                                                        Repository                                                Size
==============================================================================================================================================================================================================================================
Updating:
 buildah                                                 x86_64                                                 1.11.3-2.fc30                                                  /buildah                                                  36 M

Transaction Summary
==============================================================================================================================================================================================================================================
Upgrade  1 Package

Total size: 36 M
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : buildah-1.11.3-2.fc30.x86_64                                                                                                                                                                                               1/2 
  Cleanup    : buildah-1.11.2-2.git0bafbfe.fc30.x86_64                                                                                                                                                                                    2/2 
  Verifying  : buildah-1.11.3-2.fc30.x86_64                                                                                                                                                                                               1/2 
  Verifying  : buildah-1.11.2-2.git0bafbfe.fc30.x86_64                                                                                                                                                                                    2/2 

Updated:
  buildah.x86_64 0:1.11.3-2.fc30                                                                                                                                                                                                              

Complete!
```


**Describe the results you received:**
```
[root@93fe8a5692a4 bin]# pwd 
/usr/bin
[root@93fe8a5692a4 bin]# ls -la buildah
-rwxr-xr-x 1 root root 37519624 Oct  4 18:20 buildah
[root@93fe8a5692a4 bin]# buildah version 
buildah: symbol lookup error: buildah: undefined symbol: seccomp_api_set
```

**Describe the results you expected:**
Something like
```
buildah version
Version:         1.11.2
Go Version:      go1.12.9
Image Spec:      1.0.1
Runtime Spec:    1.0.1-dev
CNI Spec:        0.4.0
libcni Version:  
Git Commit:      
Built:           Thu Jan  1 00:00:00 1970
OS/Arch:         linux/amd64
```

**Output of `rpm -q buildah` or `apt list buildah`:**

```
rpm -q buildah
buildah-1.11.3-2.fc30.x86_64
```

**Output of `buildah version`:**

N/A, see above

**Output of `podman version` if reporting a `podman build` issue:**

```
(paste your output here)
```

**Output of `cat /etc/*release`:**

```
[root@codewind-workspaceemry865xg0d6rhrp-66cff68799-5c5zq bin]# cat /etc/*release
CentOS Linux release 7.6.1810 (Core) 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

CentOS Linux release 7.6.1810 (Core) 
CentOS Linux release 7.6.1810 (Core) 
```

**Output of `uname -a`:**

```
uname -a
Linux codewind-workspaceemry865xg0d6rhrp-66cff68799-5c5zq 3.10.0-957.12.1.el7.x86_64 #1 SMP Wed Mar 20 11:34:37 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Output of `cat /etc/containers/storage.conf`:**

```
(paste your output here)
```

**Comment by TomSweeneyRedHat** [comment]
@lsm5 any thoughts?  I don't have a CentOS vm readily available.

**Comment by lsm5** [comment]
@maysunfaisal try the centos SIG build instead. https://cbs.centos.org/koji/buildinfo?buildID=26863

**Comment by maysunfaisal** [comment]
@lsm5 downloaded the rpm from the link and it still gives me the same error:

```
[root@d46f29d39466 /]# buildah version
buildah: symbol lookup error: buildah: undefined symbol: seccomp_api_set
[root@d46f29d39466 /]# rpm -q buildah
buildah-1.11.3-2.el7.x86_64
```

**Comment by rhatdan** [comment]
Do we need a new version of libseccomp?

**Comment by maysunfaisal** [comment]
@lsm5 yes, that seemed to work

```
[root@d46f29d39466 bin]# buildah version
Version:         1.11.3
Go Version:      go1.10.2
Image Spec:      1.0.1-dev
Runtime Spec:    1.0.1-dev
CNI Spec:        0.4.0
libcni Version:  
Git Commit:      
Built:           Thu Jan  1 00:00:00 1970
OS/Arch:         linux/amd64
```

**Comment by lsm5** [comment]
updated dependency in buildah spec file. Closing...

**Comment by lsm5** [comment]
New build happening at https://cbs.centos.org/koji/taskinfo?taskID=995101 which should take care of the dependency issue.

**Comment by maysunfaisal** [comment]
@lsm5 looks like the build has failed :(

**Comment by lsm5** [comment]
Fixed https://cbs.centos.org/koji/taskinfo?taskID=995170

**Comment by pixdrift** [comment]
There appears to be a regression in the spec file and this issue has crept back into the 1.12 RPM. Looking at the spec, the libseccomp `Requires` implemented for CentOS fix is no longer there:

**Version 1.11.3 spec file:**
```
Name: %{repo}
Version: 1.11.3

...

%else
#### DO NOT REMOVE - NEEDED FOR CENTOS
Requires: libseccomp >= 2.4.1-0
Requires: container-selinux
Requires: slirp4netns >= 0.3-0
%endif
```
**Version 1.12.0 spec file:**
```
Name: %{repo}
Version: 1.12.0

...

%else
Requires: container-selinux
Requires: slirp4netns >= 0.3-0
%endif
```

Is there a source for this spec file that's publicly available?

Installing `libseccomp = 2.4.1-0.el7` resolves the issue.

**Comment by pixdrift** [comment]
@TomSweeneyRedHat you OK to re-use this issue or would you like me to create a new one?

**Comment by TomSweeneyRedHat** [comment]
@pixdrift this is fine here.

**Comment by TomSweeneyRedHat** [comment]
@lsm5 any update?

**Comment by lsm5** [comment]
@pixdrift have you tried the latest packages from OBS? See: https://github.com/containers/buildah/blob/master/install.md

**Comment by rhatdan** [comment]
Latest packages should be available.
